### PR TITLE
Create monero-core_sk.ts

### PR DESCRIPTION
**Pull request by fero-sk**
Slovak translation

### CODE REVIEW

**Comment by erciccione** [comment]
Hi, thanks for this, very appreciated. To add a new language you need to make some additional steps, at the end your Pull Request should result similar to [this one](https://github.com/monero-project/monero-gui/pull/964/files).
A line need to be added in *languages.xml*, *monero-wallet-gui.pro*, *qml.qrc* , like in the example, but adapted for Slovak. you also need to add a round flag of Slovakia.

Let me know if you need help with this (the support channel for the localizations is #monero-translations on Freenode)

**Comment by erciccione** [comment]
Hey @fero-sk you don't need to close this PR to add those changes. Just commit on the same branch. Happy to help if needed :)

**Comment by fero-sk** [comment]
I'm sorry, still learning... :) I've created a new PR.

**Comment by erciccione** [comment]
No problem. For reference: new PR is #984

File: translations/monero-core_sk.ts [hunk 1]
```diff
@@ -0,0 +1,2166 @@
...
+        <location filename="../pages/TxKey.qml" line="185"/>
+        <source>Transaction key</source>
+        <translation>Kľúč transakcie</translation>
+    </message>
+</context>
+<context>
+    <name>WizardConfigure</name>
+    <message>
+        <location filename="../wizard/WizardConfigure.qml" line="79"/>
+        <source>We’re almost there - let’
```
Comment (KanoczTomas) [inline translations/monero-core_sk.ts]: I suggest using 'Už sme skoto tam' instead of 'skoro tu'

Comment (KanoczTomas) [inline translations/monero-core_sk.ts]: missing comma before 'alebo', should be ', alebo'

**Comment by KanoczTomas** [comment]
@erciccione am new in this as well, will this be visible now that the PR is closed? Could you help with my mess? :)

**Comment by erciccione** [comment]
@KanoczTomas Thank you for the review, but this pr is deprecated, please redo your review in #984 (the active one) :)

**Comment by KanoczTomas** [comment]
Ah ok, sure ;) Will do later ypthen, I have to leave now ... thanks!

File: translations/monero-core_sk.ts [hunk 2]
```diff
@@ -0,0 +1,2166 @@
...
+    </message>
+    <message>
+        <location filename="../pages/Mining.qml" line="80"/>
+        <source>(only available for local daemons)</source>
+        <translation>(k dispozícii iba pre lokálnych démonov)</translation>
+    </message>
+    <message>
+        <location filename="../pages/Mining.qml" line="86"/>
+        <source>Mining with your computer helps strengthen the Monero network. The more that people mine, the harder it is for the network to be attacked, and every little bit helps.&lt;br&gt; &lt;br&gt;Mining also gives you a small chance to earn some Monero. Your computer will create hashes looking for block solutions. If you find a block, you will get the associated reward. Good luck!</source>
+        <translation>Ťažba s Vašim počítačom pomáha posilniť sieť Monero. Čím viac ľudí ťaží, tým je náročnejšie zaútočiť na sieť, a každý malý kúsok pomáha.&lt;br&gt; &lt;br&gt;Ťažba Vám taktiež dáva malú šancu zarobiť nejaké Monero. Váš počítač bude vytvárať haše hľadaním riešenia v bloku. Ak nájdete blok, dostanete s tým súvisiacu odmenu. Veľa šťastia!</translation>
```
[the remark this answers is not in the document]
Comment (fero-sk) [reply translations/monero-core_sk.ts]: Hello, the word "haš" is used here https://sk.wikipedia.org/wiki/Ha%C5%A1ovacia_funkcia

File: translations/monero-core_sk.ts [hunk 3]
```diff
@@ -0,0 +1,2166 @@
...
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="135"/>
+        <source>Transaction priority</source>
+        <translation>Priorita transakcie</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="175"/>
+        <source>All</source>
+        <translation>Všteko</translation>
```
[the remark this answers is not in the document]
Comment (fero-sk) [reply translations/monero-core_sk.ts]: Corrected.

File: translations/monero-core_sk.ts [hunk 4]
```diff
@@ -0,0 +1,2166 @@
...
+        <location filename="../pages/Transfer.qml" line="716"/>
+        <source>Wallet is not connected to daemon.</source>
+        <translation>Peňaženka nie je pripojená k démonovi.</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="719"/>
+        <source>Connected daemon is not compatible with GUI. 
+Please upgrade or connect to another daemon</source>
+        <translation>Pripojený démon nie je kompatibilný s GUI. 
+Prosím aktualizujte alebo pripojte k inému démonovi</translation>
```
[the remark this answers is not in the document]
Comment (fero-sk) [reply translations/monero-core_sk.ts]: Corrected.

File: translations/monero-core_sk.ts (hunk 1 above)
Comment (fero-sk) [reply translations/monero-core_sk.ts]: Corrected.

File: translations/monero-core_sk.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (KanoczTomas) [reply translations/monero-core_sk.ts]: Hello, I did not see it used like that in academic crypto field. Wiki is not the best source, but other more trustworthy sources prove your point, e.g. http://www.lexman.sk/blog/2015/existuje-slovenske-slovo-has/. So your form is correct and I know a new thing ;) I suppose a win-win.

[the remark this answers is not in the document]
Comment (fero-sk) [reply translations/monero-core_sk.ts]: I know, that it's not used. Maybe "odtlačok" would be better... Don't know...
The word "seed" I left as is in english.

**Comment by erciccione** [comment]
At this point I guess it's too late, but you should discuss on the active PR, not this one, or who will review later will not understand where are those changes from

**Pull request closed without merging**